### PR TITLE
Update 16-spectro-downloads.md

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR removes the 3.3.0 Edge URL that has been flagged with a bug. The customer should use the latest 3.3.2 installer instead. 

## Review Changes

💻 [Add Preview URL]()

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1259#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/16-spectro-downloads.md [hunk 1]
```diff
@@ -91,12 +90,10 @@ v3.3.0 |  [Linux-amd64](https://software.spectrocloud.com/stylus/v3.3.0/cli/linu
 |Version|Docker Image| Optical Disk Image (ISO) |
 |---|---|--|
 |3.3.2     | `gcr.io/spectro-images-public/stylus:v3.3.2`| [ISO](https://software.spectrocloud.com/stylus/v3.3.2/stylus-v3.3.2-amd64.iso) |
-|3.3.0     | `gcr.io/spectro-images-public/stylus:v3.3.2`| [ISO](https://software.spectrocloud.com/stylus/v3.3.0/stylus-v3.3.0-amd64.iso) |
```
Comment (ritawatson) [inline content/docs/16-spectro-downloads.md:94]: Should this say v3.3.0?

**Review by ritawatson — APPROVED** [review]
One correction may be needed. Otherwise, LGTM.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1259#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1259#pullrequestreview-1400530877" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/16-spectro-downloads.md (hunk 1 above)
Comment (karl-cardenas-coding) [reply content/docs/16-spectro-downloads.md:94]: @ritawatson I see. The good news is that line is going away 😄

**Pull request merged**